src/MastodonConnectionHelper.cs

```diff
@@ -36,11 +37,31 @@ public static async Task<List<string>> GetPinnedTagsAsync()
         await conn.OpenAsync();
 
         var res = new List<string>();
-        await using var cmd = new NpgsqlCommand("SELECT DISTINCT col->'params'->>'id' FROM web_settings, json_array_elements(data->'columns') col WHERE col->>'id' = 'HASHTAG' AND col->'params'->>'id' IS NOT NULL ORDER BY col->'params'->>'id' ASC", conn);
+        // Column 0: the 'original' tag with was pinned
+        // Column 1: Config of 'Include additional tags for this column' this includes the the tags in 'any' array.
+        await using var cmd = new NpgsqlCommand("SELECT DISTINCT col->'params'->>'id', col->'params'->>'tags' FROM web_settings, json_array_elements(data->'columns') col WHERE col->>'id' = 'HASHTAG' AND col->'params'->>'id' IS NOT NULL ORDER BY col->'params'->>'id' ASC", conn);
         await using var reader = await cmd.ExecuteReaderAsync();
         while (await reader.ReadAsync())
+        {
             res.Add(reader.GetString(0));
+            if (! reader.IsDBNull(1))
+            {
+                JsonDocument doc = JsonDocument.Parse(reader.GetString(1));
 
+                JsonElement anyArray = doc.RootElement.GetProperty("any");
+                foreach (JsonElement item in anyArray.EnumerateArray())
+                {
+                    JsonElement value = item.GetProperty("value");
+                    if (value.ValueKind != JsonValueKind.Null)
+                    {
+                        String? valuestring = value.GetString();
+                        if (! valuestring.IsNullOrEmpty())
```

Comment:
use `var` and  `HasValue()`

```suggestion
                var doc = JsonDocument.Parse(reader.GetString(1));

                var anyArray = doc.RootElement.GetProperty("any");
                foreach (var item in anyArray.EnumerateArray())
                {
                    var value = item.GetProperty("value");
                    if (value.ValueKind != JsonValueKind.Null)
                    {
                        var valuestring = value.GetString();
                        if (valuestring.HasValue())
```